### ISSUE TITLE
Playlist track doesn't fade-in

### ISSUE THREAD

**Issue by rmasoni** [MaterialFoundry/MaterialCompanion#12]
When I click a button on the StreamDeck app for iPadOS to play a track from a Playlist, it starts the track from 0:00, but without the fade-in. Fade-out works as expected when I click the button again to stop the track.

I'm using Material Companion for macOS ARM64. Foundry VTT 12 Build 331 (node client) on Brave, no other Playlist-related modules.

Here's my setup:

![ScreenShot 2024-11-21 at 20 05 04@2x](https://github.com/user-attachments/assets/741b7d4e-2307-4314-bf62-eee2e82beac8)

![ScreenShot 2024-11-21 at 20 05 23@2x](https://github.com/user-attachments/assets/5d2ebdee-7631-44d8-b6c6-5babbb3a847e)

![ScreenShot 2024-11-21 at 20 05 40@2x](https://github.com/user-attachments/assets/dbc6de60-a137-4fb3-92d9-f779bd4ab63e)

**Comment by MaterialFoundry** [comment]
Whether or not this is a bug depends on which fade you've configured.
-Playlist fade appears to only work as a fade-out in Foundry, whether you use Material Deck or a mouse. If you want it to also work as a fade-in you should do a feature request on the [Foundry repository](https://github.com/foundryvtt/foundryvtt).
-Sound fade works as both a fade-in and fade-out, and for me it also works with Material Deck.

Btw, this repository is meant for Material Companion related issues, so I'm going to close this issue. If you still believe this is a Material Deck issue, or if there's anything else Material Deck related, please head over to the [Material Deck repository](https://github.com/materialfoundry/materialdeck).

**Comment by rmasoni** [comment]
Sorry, I didn't realize there was a more appropriate repository to report this.

After speaking with a moderator in Foundry's Discord, we came to the conclusion that this is a recent bug, as fade-in worked properly until at least October (last time I played).

Luckily, there's a workaround: if you set the Fade Duration in the Sound instead of the Playlist, it works properly.

I created this [issue](https://github.com/foundryvtt/foundryvtt/issues/11856). Let's hope they can fix it soon.
